keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -982,46 +993,209 @@ you need any help or guidance.
+that might indicate a serious problem?
+-->
+
+The AdminNetworkPolicy API has a `Status` field which should be used by the
+implementation to report weather or not the rules were correctly programmed.
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+<!--
+Describe manual testing that was done and the outcomes.
+Longer term, we may want to require automated upgrade/rollback tests, but we
+are missing a bunch of machinery and tooling and can't do that now.
+-->
+
+This will be tested once implementations of the API have been completed.
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+<!--
+Even if applying deprecation policies, they may still surprise some users.
+-->
+
+No.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
+
+Since the controller for this feature will not be implemented in-tree, it will be
+the responsibility of the implementations to report metrics as they see fit.
+
+###### How can someone using this feature know that it is working for their instance?
+
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
+ 
+- [x] API .status
+  - Condition name: The Condition name will not be standardized in `alpha` however
+  implementations are given the `status` field to report what they see fit.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+If an AdminNetworkPolicy is applied the policy rules should be applied and enforced
+as fast as possible to ensure the cluster is always in a secure state. 
+
```

Comment:
This doesn't add much. Presumably we can't dictate this to implementations, so you might want to add something to the effect that "specific SLOs will be defined by the implementations".

Reply:
Will do!

Reply:
done